jobbergate-agent/Makefile

```diff
@@ -0,0 +1,40 @@
+SHELL:=/bin/bash
+ROOT_DIR:=$(shell dirname $(shell pwd)).PHONY: help
+
+.PHONY: lint
+lint: ## Run linter
+	tox -e lint
+
+.PHONY: format
+format: ## Run formatter
+	tox -e format
+
+.PHONY: get-lib
+get-lib: ## Get lib dependency
+	@charmcraft fetch-libs
+
+.PHONY: static
+static: get-lib ## Run static analysis
+	tox -e static
+
+.PHONY: qa
+qa: lint static ## Run all quality checks
+
+.PHONY: charm
+charm: ## Pack the charm
```

Comment:
question: shouldn't it run `get-lib` first?

Reply:
I tested it out. It seems that when you pack the charm, it automatically fetches the libs it needs, so we shouldn't need to add that here.